chirp/drivers/icm710.py

```diff
@@ -0,0 +1,178 @@
+# Copyright 2024 Paul Decker <kg7hf1@gmail.com>
+#      Derives from template.py
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import logging
+from chirp import chirp_common, directory
+from chirp import bitwise
+from chirp.drivers import icf
+
+LOG = logging.getLogger(__name__)
+
+BAUDS = [4800]
+POWER_LEVELS = []
+DUPLEX_MODES = ['split', 'off']
+# do not reorder this list
+MODULATION_MODES = ["USB", "R3E", "AM", "LSB", "AFS", "FSK", "CW"]
+TUNING_STEPS = [0.5]
+
+MEM_FORMAT = """
+
+#seekto 0xE020;
+struct {
+  bbcd rx_freq[4];
+  u8 mode;
+  char name[7];
+  bbcd tx_freq[4];
+}memory[232];
+
+"""
+
+
+@directory.register
+class IcomM710Radio(icf.IcomCloneModeRadio):
+    """ICOM IC-M710"""
+    VENDOR = "Icom"
+    MODEL = "IC-M710"
+    BAUD_RATE = 4800
+
+    _model = "\x162\x00\x01"  # 4-byte mode string
+    _endframe = ''
+    _ignore_clone_ok = True
+
+    _memsize = 0xEFC0
+    _ranges = [(0xE000, 0xEFC0, 32)]
+    _can_hispeed = False
+    _double_ident = False
+
+    _raw_frames = False
+    _highbit_flip = False
+
+    # Return information about this radio's features, including
+    # how many memories it has, what bands it supports, etc
+    def get_features(self):
+        rf = chirp_common.RadioFeatures()
+        rf.has_bank_index = False
+        rf.has_dtcs = False
+        rf.has_rx_dtcs = False
+        rf.has_dtcs_polarity = False
+        rf.has_mode = True
+        rf.has_offset = False
+        rf.has_name = True
+        rf.has_bank = False
+        rf.has_bank_names = False
+        rf.has_tuning_step = False
+        rf.has_ctone = False
+        rf.has_cross = True
+        rf.has_infinite_number = False
+        rf.has_nostep_tuning = True
+        rf.has_comment = True
```

Comment:
This should be `False` since you're not able to store a user's comment. Setting it to `False` means chirp will just handle it for you at the file layer.